package.json

```diff
@@ -152,9 +153,13 @@
     "neo4j-driver-bolt-connection@npm:5.20.0": "patch:neo4j-driver-bolt-connection@npm%3A5.20.0#~/.yarn/patches/neo4j-driver-bolt-connection-npm-5.20.0-1f7809f435.patch",
     "neo4j-driver-core@npm:5.20.0": "patch:neo4j-driver-core@npm%3A5.20.0#~/.yarn/patches/neo4j-driver-core-npm-5.20.0-99216f6938.patch",
     "@apollo/server-plugin-landing-page-graphql-playground": "npm:empty-npm-package@*",
+    "@apollo/server/express": "npm:empty-npm-package@*",
```

Comment:
Trying to understand why

Reply:
`@apollo/server` requires/depends on `express`. This is for BC.
I don't want express being pulled in, so this swaps it to an empty package.